docs/releases/overview.md

```diff
@@ -26,19 +26,57 @@ See the full release notes on [GitHub ↗](https://github.com/vegaprotocol/vega/
+
+#### Changing clef address now requires re-importing config
+The Nodewallet.ETH section of the config has been removed, and as a consequence some CLI arguments have changed when using clef. Before, when starting a Vega node with a clef wallet, Vega would read whatever clef address was in nodewallet.ETH, whereas after this change, the network only ever uses the value set for the clef address when the key was imported/generated. This work was done under issue [6291 ↗](https://github.com/vegaprotocol/vega/issues/6291)
+
+#### Wallet v2 API `session` renamed
+To add more clarity to what the wallet API does, the `session` namespace has been renamed to `client`. This work was done under issue [6314 ↗](https://github.com/vegaprotocol/vega/issues/6314)
+
+#### New features: Core
+
+#### Block explorer APIs
+In order to support the block explorer being able to list transactions and provide a good user experience around searching with transaction ids, a service and new APIs have been implemented. Using the Postgres Tendermint indexer was resulting in datanode stability issues and in some cases affecting core node stability. This new service and APIs will allow good user experiences to be built in front end block explorer applications. This work was carried out in [6163 ↗](https://github.com/vegaprotocol/vega/issues/6163)
+
+#### New features: Data node
+
+#### Datanode snapshots
+As part of the protocol upgrade process and when new nodes join the network, the nodes need to ensure the datanode data is correct. Datanode snapshots will be used for these use cases. This initial implementation allows creation of deterministic delta snapshots and the loading up a datanode from the snapshot data. This is an ongoing in-development feature, this part was implemented in 
+
+#### Update ssset proposal to include the asset ID being updated in GraphQL
+In order to ensure GraphQL users understand which asset is being updated the field `assetId` has been added into the `UpdateAsset` proposals response. This work was done under issue [6296 ↗](https://github.com/vegaprotocol/vega/issues/6296)
+
+#### Add rate limiter for GraphQL
+During testing it was identified that GraphQL subscriptions were able to cause overloading on the datanode. To avoid this a rate limiter has been implemented for GraphQL subscriptions. This work was implemented under [6334 ↗](https://github.com/vegaprotocol/vega/pull/6334)
+
+#### New features: Wallet
+
+#### Add commit hash to verison if is development version
+To avoid version confusion by developers and builders due to the wallet not raising compatibility issues between different development versions of the software. The version check has been enhanced to add the commit hash (first 8 characters) behind the +dev build tag. This work was carried out in issue [6283 ↗](https://github.com/vegaprotocol/vega/issues/6283)
```

Comment:
```suggestion
To avoid version confusion by developers and builders due to the wallet not raising compatibility issues between different development versions of the software, the version check has been enhanced to add the commit hash (first 8 characters) behind the +dev build tag. This work was carried out in issue [6283 ↗](https://github.com/vegaprotocol/vega/issues/6283)
```